### PR TITLE
LPS-180478 | Master

### PR DESCRIPTION
**Pull request by Hanlf**
https://liferay.atlassian.net/browse/LPS-180478

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileOrder.testcase [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+			locator1 = "LexiconTable#TABLE_ROW",
+			value1 = "DM Document Title 1");
+
+		AssertTextEquals.assertPartialText(
+			index = 2,
+			locator1 = "LexiconTable#TABLE_ROW",
+			value1 = "DM Document Title 2");
+	}
+
+	@description = "This test ensures that when setting the DM File Order at different levels(Instance and System settings), the instance settings will take effect."
```
Comment (brianwulbern) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileOrder.testcase:96]: ...`levels (Instance`...

^ the description can use a space here, it's currently missing

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileOrder.testcase [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+			locator1 = "LexiconTable#TABLE_ROW",
+			value1 = "DM Document Title 2");
+
+		AssertTextEquals.assertPartialText(
+			index = 2,
+			locator1 = "LexiconTable#TABLE_ROW",
+			value1 = "DM Document Title 1");
+	}
+
+	@description = "This test ensures that when setting the DM File Order at different levels(Site and Instance settings), the site settings will take effect."
```
Comment (brianwulbern) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileOrder.testcase:181]: ...`levels(Site`...

@Hanlf same here

**Review by austinchiang — COMMENTED** [review]
Same comments as Brian, besides that LGTM

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileOrder.testcase (hunk 1 above)
Comment (Hanlf) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileOrder.testcase]: updated

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileOrder.testcase (hunk 2 above)
Comment (Hanlf) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileOrder.testcase]: Updated, thanks!

**Comment by Hanlf** [comment]
@brianwulbern @austinchiang updated, please review.

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**